lib/pymedphys/_mosaiq/mock/extract_from_bak.py

```diff
@@ -49,11 +49,11 @@ def extract(
     if sa_password is None:
         sa_password = os.getenv("MSSQL_SA_PASSWORD", default=secrets.token_urlsafe())
 
-    # start_mssql_docker_image_with_bak_restore(
-    #     bak_filepath=bak_filepath,
-    #     sa_password=sa_password,
-    #     database_name=database_name,
-    # )
+    start_mssql_docker_image_with_bak_restore(
```

Comment:
the extract method needs docstring so it's obvious that one needs to pass in an appropriate database name.
The database_name should not have a default, or if it does have one, it shouldn't be PRACTICE (in general, the database name is "MOSAIQ"...)

Reply:
In the follow up PR I didn't even end up using this function.  Let's leave it here as it may have utility, but let's not focus on it too much more for now.  (I definitely need to be [80/20 ing this](https://asana.com/resources/pareto-principle-80-20-rule))